### ISSUE TITLE
Errno:2 "Failed to copy over our resources to RoofFS"

### ISSUE THREAD

**Issue by MattieThurlow96** [pwn20wndstuff/Undecimus#128]
This happens since the update to 2.0.0.
iOS 11.4b3, iPhone 8 Plus

**Comment by SanZuev** [comment]
Hi. I am on iPhone 6 iOS 11.3.1. I use unc0ver sice its first public beta (RC5). everything went fine, i succeeded to update on the first place, but ran into respring loop, as i though it was a major update and could conflict with the previous one (1.1.4), i restored RootFS. After that i got errno 0 (it was the first public pre-release of unc0ver 2.0.0). The description mentioned "unable to extract bootstrap". Then I upgraded to the second release of unc0ver v2.0.0 and I got errno:2 "Failed to copy over our resources to RoofFS". This has never happened before. I tried restoring with Rollectra and JB Remover by pwned4ever. No effect. 
![image0](https://user-images.githubusercontent.com/45495651/49306517-339e2580-f4f4-11e8-8f6b-5784a388f9ba.png)
Hope it helps.
Thanks
